README.rst

```diff
@@ -311,7 +311,7 @@ Testing
 To run the tests, repeat the
 ``postgres createuser && postgres createdb`` commands as above with the
 values for the {user, password, database} as defined in the
-``TestConfig`` class within ``portal.config.py``
+``BaseConfig`` class within ``portal\config\config.py``
```

Comment:
For `Testing`, I do believe it's in the ``TestConfig``:

```
    SQLALCHEMY_DATABASE_URI = testing_sql_url()
```

Reply:
Updated